test/dotnet-publish.Tests/PublishAppWithBuildDependency.cs

```diff
@@ -17,8 +17,6 @@ public void PublishExcludesBuildDependencies()
             var testInstance = TestAssetsManager.CreateTestInstance("AppWithDirectDependencyAndTypeBuild")
                 .WithLockFiles();
 
-            new RestoreCommand() { WorkingDirectory = testInstance.TestRoot }.Execute().Should().Pass();
```

Comment:
Looking at the test I'm not sure why it would be unless there's an issue with using the built Microsoft.NETCore.App